components/badge/badge_eink.h

```diff
@@ -57,37 +59,52 @@ struct badge_eink_update {
  */
 extern void badge_eink_update(const uint32_t *buf, const struct badge_eink_update *upd_conf);
 
-/* badge_eink_display 'mode' settings */
-// bitmapped flags:
-#define DISPLAY_FLAG_8BITPIXEL  1
-#define DISPLAY_FLAG_ROTATE_180 2
-#define DISPLAY_FLAG_NO_UPDATE  4
-#define DISPLAY_FLAG_FULL_UPDATE 8
-// fields and sizes:
-#define DISPLAY_FLAG_LUT_BIT    8
-#define DISPLAY_FLAG_LUT_SIZE   4
+/** badge_eink_display 'mode' settings */
+enum badge_eink_flags_t
+{
+	// bitmapped flags:
+	DISPLAY_FLAG_8BITPIXEL   =  1,
+	DISPLAY_FLAG_ROTATE_180  =  2,
+	DISPLAY_FLAG_NO_UPDATE   =  4,
+	DISPLAY_FLAG_FULL_UPDATE =  8,
+
+	// fields and sizes:
+	DISPLAY_FLAG_LUT_BIT     =  8,
+	DISPLAY_FLAG_LUT_SIZE    =  4,
```

Comment:
This is weird to me: an enum implies a collection of discrete, individual options. 'flags', on the other hand, implies a bitmask of individual settings that can be `or`ed together.

Though probably the compiler accepts it, I wouldn't consider '5' to be a valid value of type `enum badge_eink_flags_t` (as it doesn't appear in the enumeration), but I would consider `DISPLAY_FLAG_8BITPIXEL|DISPLAY_FLAG_NO_UPDATE` a valid value for `int`-typed flags